### DISCUSSION TITLE
zigbee2mqtt non funziona

### DISCUSSION THREAD

**Discussion by Olodumdomotico** [indomus/forum#150] [ZigBee, Z-Wave, Thread, Matter]
Salve ho Home Assistant versione OS direttamente su SD raspberry
Ho installato chiavetta conbee2 ,configurata correttamente 

Poi ho creato account MQTT con suo user e password (non amministratore)
Installato "mosquitto broker" ed avviato senza fare modifiche in configurazione
Impostazioni->integrazioni -> e configurato MQTT con user e password dell'account MQTT
Fin qui sembra tutto ok

Poi installato zigbee2mqtt  , ma non riesco a farlo funzionare, ho visto molte guide e non capisco dove sbaglio
Ho inserito user e pass dell'account MQTT ed alla voce "port" ho provato sia ttyAMA0 che il codice seriale
[[![Screenshot-7.png](https://i.postimg.cc/VNM9bMX9/Screenshot-7.png)](https://postimg.cc/JyrHfs6t)](url) configurazione (ho provato anche con ID seriale
[[![Screenshot-6.png](https://i.postimg.cc/hG9xGnqp/Screenshot-6.png)](https://postimg.cc/PL5N6BJw)](url) questi sono gli errori

Qualcuno può aiutarmi , sono 2 giorni che faccio prove

**Comment by KC-inDomus** [comment]
è sbagliata la config della chiavetta. guarda bene la guida sul sito.

**Comment by Olodumdomotico** [comment]
Strano ,deconz funziona con questo ID
/dev/serial/by-id/usb-dresden_elektronik_ingenieurtechnik_GmbH_ConBee_II_DE2424377-if00
mentre zigbee2mqtt lo inserisco alla voce "port" ma non parte

ho inserito e tolto "adapter: deconz" , ma niente

**Reply by snakuzzo** [reply]
Ma hai letto il log ?
Il container (addon) di zigbee2mqtt non vede la chiavetta sulla porta `/dev/ttyACM1`
In pratica non viene proprio montata lì

**Comment by Olodumdomotico** [comment]
in effetti nei log non accetta la destinazione della chiavetta
Ma è lo stesso ID scritto su deconz che funziona correttamente .......
[![Screenshot-8.png](https://i.postimg.cc/G3JdPJYM/Screenshot-8.png)](https://postimg.cc/F1RMNSTc)[](url)

**Comment by Olodumdomotico** [comment]
Il log del post sopra è riferito a questo PORT
/dev/serial/by-id/usb-dresden_elektronik_ingenieurtechnik_GmbH_ConBee_II_DE2424377-if00
Che sono convinto sia corretto........ però non riesco ugualmente

**Reply by snakuzzo** [reply]
Sì ma nel log c'è scritto `/dev/ttyACM1` ... è quello che conta

**Reply by snakuzzo** [reply]
Non vorrei sia un problema dell'addon

**Comment by Olodumdomotico** [comment]
Io parto dal presupposto che deconz funziona a quel indirizzo e visualizza già alcuni dispositivi zigbee
Non essendo un esperto, mi serve qualche idea perché non so più cosa provare

**Reply by snakuzzo** [reply]
Sì ma rispondi sotto lo stesso thread, non crearne ogni volta uno nuovo.

Es.

![image](https://user-images.githubusercontent.com/7205371/114553984-83fe2400-9c66-11eb-8888-9d57c02810e8.png)

**Reply by Olodumdomotico** [reply]
Ho creato altro thread? Scusa probabilmente da cell ho fatto confusione

**Reply by Olodumdomotico** [reply]
può essere che su questo plugin  l'utente su cui eseguo Zigbee2MQTT non abbia accesso in scrittura alla porta ??
Sto dicendo una cavolata?? questa cavolata mi fa impazzire, è semplicissimo, ma mi funziona :(

**Comment by sebyno** [comment]
Forse dico una stupidata ma non è che hai deconz attivo e che quindi "occupa" la risorsa e di conseguenza Zigbee2MQTT non riesce a usare la stessa porta?

**Reply by Olodumdomotico** [reply]
Deconz è attivo, non credo sia un problema, stasera dopo il lavoro provo
Grazie

**Reply by snakuzzo** [reply]
Ma NON deve essere attivo Deconz!
O usi Deconz o Zigbee2mqtt. Non puoi usare la chiavetta con entrambi nello stesso momento :D

**Reply by Olodumdomotico** [reply]
Ero convinto che andavano abilitati entrambi, grazie, stasera faccio sapere esito :D

**Reply by Olodumdomotico** [reply]
Effettivamente ora funziona !!!!!!!
Purtroppo nella mia testa ero convinto che entrambi potevano funzionare contemporaneamente.

Comunque la prova era per cercare di fare il pairing con i devio Vimar, ma al momento non ci sono riuscito..........sto cercando di eliminare il gateway
